### PR TITLE
[doc](flink-connector) update flink doc and options

### PR DESCRIPTION
**Pull request by JNSimba**
## Proposed changes

Issue Number: close #xxx

update flink doc and options

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by zy-kkk — DISMISSED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Review by caoliang-web — APPROVED** [review]
LGTM

File: docs/zh-CN/docs/ecosystem/flink-doris-connector.md [hunk 1]
```diff
@@ -312,16 +313,18 @@ ON a.city = c.city
...
-| doris.request.retries            | 3             | N        | 向 Doris 发送请求的重试次数                                                                                  |
-| doris.request.connect.timeout.ms | 30000         | N        | 向 Doris 发送请求的连接超时时间                                                                                |
-| doris.request.read.timeout.ms    | 30000         | N        | 向 Doris 发送请求的读取超时时间                                                                                |
+| jdbc-url                         | --            | N        | jdbc连接信息，如: jdbc:mysql://127.0.0.1:9030                |
+| table.identifier                 | --            | Y        | Doris 表名，如：db.tbl                                       |
+| username                         | --            | Y        | 访问 Doris 的用户名                                          |
+| password                         | --            | Y        | 访问 Doris 的密码                                            |
+| auto-redirect                    | false         | N        | 是否重定向StreamLoad请求。开启后StreamLoad将通过FE写入，不再显示获取BE信息，同时也可通过开启该参数写入SelectDB Cloud |
```
Comment (DongLiang-0) [inline docs/zh-CN/docs/ecosystem/flink-doris-connector.md:324]: Same as above

File: docs/en/docs/ecosystem/flink-doris-connector.md [hunk 2]
```diff
@@ -309,16 +310,18 @@ ON a.city = c.city
...
-| doris.request.read.timeout.ms    | 30000         | N        | Read timeout for sending requests to Doris                                                                                                              |
+| jdbc-url                         | --            | N        | jdbc connection information, such as: jdbc:mysql://127.0.0.1:9030 |
+| table.identifier                 | --            | Y        | Doris table name, such as: db.tbl                            |
+| username                         | --            | Y        | username to access Doris                                     |
+| password                         | --            | Y        | Password to access Doris                                     |
+| auto-redirect                    | false         | N        | Whether to redirect StreamLoad requests. After being turned on, StreamLoad will be written through FE, and BE information will no longer be displayed. At the same time, it can also be written to SelectDB Cloud by turning on this parameter. |
```
Comment (DongLiang-0) [inline docs/en/docs/ecosystem/flink-doris-connector.md:321]: Isn’t it a bit inappropriate to put Selectdb Cloud on doris official website?

**Review by DongLiang-0 — APPROVED** [review]
LGTM

**Pull request merged**